client/components/domains/register-domain-step/index.jsx

```diff
@@ -796,6 +797,12 @@ class RegisterDomainStep extends React.Component {
 	};
 
 	handleSubdomainSuggestions = ( domain, timestamp ) => subdomainSuggestions => {
+		subdomainSuggestions = subdomainSuggestions.map( suggestion => {
+			suggestion.fetch_algo = endsWith( suggestion.domain_name, '.wordpress.com' )
+				? '/domains/search/wpcom/20180816'
```

Comment:
Is hardcoding the data a sustainable solution for you? When do you foresee changing it? I imagine it would be too much overhead to change it everytime you start a new A/B test? Could we use the A/B test name instead so that at least you'd only have to search and replace that? I am afraid that date could soon become meaningless if there's no clear and easy process when and how to update it. I'd rather not have it at all than have outdated values in the data as soon as we forget to update this. All TrainTracks data are timestamped anyway, so we could in principle track the configuration that was used already.